city_scrapers/spiders/il_metra_board.py

```diff
@@ -54,7 +55,14 @@ def _parse_classification(self, item):
         """
         Parse or generate classification (e.g. public health, education, etc).
         """
-        return 'transit'
+        full_name = item.css('td[headers=Name]::text').extract_first()
+
+        if "Board Meeting" in full_name:
+            return "Board Meeting"
+        elif "Citizens Advisory Board Meeting" in full_name:
+            return "Citizens Advisory"
+        else:
+            return ''
```

Comment:
@ab1470 this is the right idea :) However, since "Board Meeting" is a substring of "Citizens Advisory Board Meeting", line 63 will never run and no meetings will be classified as "Citizens Advisory".

I also see that there is a safety committee meeting posted for July. Let's classify that one as a "Commitee Meeting". 

Almost there! :)